internal/patroni/config.go

```diff
@@ -450,6 +450,19 @@ func instanceEnvironment(
 			Name:  "PATRONICTL_CONFIG_FILE",
 			Value: configDirectory,
 		},
+		// This allows a custom CA certificate to be mounted for Postgres LDAP
+		// authentication via spec.config.files.
+		// - https://wiki.postgresql.org/wiki/LDAP_Authentication_against_AD
+		//
+		// When TLS_CACERT is used as an environment variable rather 'LDAP' is
```

Comment:
Rather? Not sure I understand this wording.

Reply:
⛏️ This is the only things that jumped out at me as well.  And might be worth noting that the Environment Variables section of that doc does spell out the proper syntax for OpenLDAP env vars:

```
LDAP<option-name>
              Set <option-name> as from ldap.conf
```

Reply:
I'll work on rewording that comment, looks like I missed this during some refactoring.

Reply:
Updated. Thank you both for the catch.